File: py/db/template/java/model_v2.java
package {{package_name}}.model;


import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import java.io.Serial;
import java.io.Serializable;
import lombok.Data;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.NotEmpty;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.NotEmpty;

/**
* @author {{author}}
* @since {{today}}
*/
@Data
@ApiModel(value="{{table_comment}}")
public class {{class_name}} implements Serializable {

    @Serial
    private static final long serialVersionUID = -1L;

    {{fields}}

}

File: py/db/template/kotlin/model_v2.kt
package {{package_name}}.model

import java.io.Serial
import java.io.Serializable
import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import javax.validation.constraints.NotNull
import javax.validation.constraints.NotEmpty

@ApiModel(title="{{table_comment}}")
class {{class_name}}:Serializable {

    {{fields}}

    companion object {
        @Serial
        private const val serialVersionUID: Long = -1L
    }

}